collectors/ioping.plugin/README.md

```diff
@@ -1,86 +1,120 @@
 <!--
 title: "ioping.plugin"
 custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/ioping.plugin/README.md
+description: "Netdata's ioping plugin allows you to measure your disk latency."
 -->
 
 # ioping.plugin
 
-The ioping plugin supports monitoring latency for any number of directories/files/devices,
-by pinging them with `ioping`.
+The ioping plugin supports monitoring latency for any number of directories/files/devices, by pinging them with `ioping`.
+When your disks don't respond in the time you defined in `ioping.conf`, the plugin raises an alert (`ioping_disk_latency`).
 
-A recent version of `ioping` is required (one that supports option `-N`).
-The supplied plugin can install it, by running:
+The collector is enabled if Netdata detects that you have `ioping` installed. Otherwise, it is disabled. 
 
-```sh
-/usr/libexec/netdata/plugins.d/ioping.plugin install
-```
+## Prerequisites
+
+To use this plugin, you need to install `ioping`.
+
+## Installing ioping
+
+You can install `ioping` using the plugin.
+The following command will download, build and install the correct `ioping` version as `/usr/libexec/netdata/plugins.d/ioping`.
+
+To install `ioping`, run:
+
+   ```sh
+   /usr/libexec/netdata/plugins.d/ioping.plugin install
+   ```
+   Once `ioping` is installed, Netdata will automatically start collecting disk latency data.
+
+> Note: If you set up Netdata using a different environment path than the default (`/etc/netdata/.environment`), use the option `-e` to indicate where the Netdata environment file is installed.
+
+## Configuring ioping.plugin
+
+Netdata ships each plugin and collector with a default configuration. To determine whether you need to make changes, check the [default configuration for `ioping.conf`](https://raw.githubusercontent.com/netdata/netdata/master/health/health.d/ioping.conf).
 
-The `-e` option can be supplied to indicate where the Netdata environment file is installed. The default path is `/etc/netdata/.environment`.
+To make changes to the ioping.plugin configuration: 
 
-The above will download, build and install the right version as `/usr/libexec/netdata/plugins.d/ioping`.
+1. Navigate to the [Netdata config directory](/docs/configure/nodes#the-netdata-config-directory):
 
-Then you need to edit `/etc/netdata/ioping.conf` (to edit it on your system run
-`/etc/netdata/edit-config ioping.conf`) like this:
+   ```bash
+   cd /etc/netdata
+   ``` 
+
+2. Use the [`edit-config`](/docs/configure/nodes#use-edit-config-to-edit-configuration-files) script to edit `ioping.conf`.
+
+   ```bash
+   sudo ./edit-config ioping.conf
+   ```
+ ### Example
 
 ```sh
-# uncomment the following line - it should already be there
+# Uncomment the following line
 ioping="/usr/libexec/netdata/plugins.d/ioping"
```

Comment:
Let's remove the line. The [default value](https://github.com/netdata/netdata/blob/31c4623cdf062b29675b612391d77b2f7f572a41/collectors/ioping.plugin/ioping.plugin.in#L163) is correct regardless of the install prefix.

Reply:
Thanks for the info.